web/apps/serve.go

```diff
@@ -248,17 +248,17 @@ func ServeAppFile(c echo.Context, i *instance.Instance, fs appfs.FileServer, web
 				return renderMovedLink(c, i, to, subdomainType)
 			}
 		}
+	}
```

Comment:
Shouldn't logged-in user have direct access to their password-protected sharings instead?

Reply:
No, the user may want to check that the link they will give is really protected by a password. And they can still access their data via the normal way.